napalm/base/base.py

```diff
@@ -28,8 +28,49 @@
 from napalm.base.exceptions import ConnectionException
 
 
+yang_filters = {
```

Comment:
I was just wondering, would it be possible to make this more dynamic and extensible in your own environment - without a fork, that is.

Reply:
This is basically our own implementation of "getters" so you could say it's our "business logic". So "no", I am afraid this can't be generated dynamically.